package.json

```diff
@@ -19,6 +19,7 @@
     "chalk": "^2.4.1",
     "commander": "^2.20.3",
     "cron": "^1.4.1",
+    "crypto-js": "^4.0.0",
```

Comment:
v4 explicitly avoided using math.random (see release notes)

Reply:
seems like there is a ticket why, but it is probably safer to use it

Reply:
@ibroom can you elaborate on your comment? It isn't clear what action you are suggesting.

Reply:
updated previous comment.